devmand/gateway/src/devmand/test/cli/StructuredUbntDeviceTest.cpp

```diff
@@ -420,6 +420,8 @@ static const string EXPECTED_OUTPUT =
 
 TEST_F(StructuredUbntDeviceTest, DISABLED_getState) {
   devmand::Application app;
+  app.init();
```

Comment:
We cannot rely on app.init here. 
Instead, inject the mregistry into the device via constructor (additional constructor overload might be needed)

Reply:
We do not rely on app.init, we rely that the CliEngine will have mregistry (which must hold true as it does for other singleton objects like executors). app.init is just a means to get CliEngine initialized.

Maybe we could inject engine directly as has been done for some other tests.